src/blahp/src/CMakeLists.txt

```diff
@@ -63,6 +63,10 @@ add_library(libblahp STATIC
 	server.c
 )
 
+# server.c defines a bunch of callbacks that don't need their args
+# disable warnings for those
+set_source_files_properties(server.c PROPERTIES COMPILE_FLAGS -Wno-unused-parameter)
```

Comment:
I presume that `-Wno-unused-parameter` will be removed when `server.c` is actually fixed.